src/styles/themes/gothamist/_theme.overrides.scss

```diff
@@ -951,9 +951,7 @@ form .o-button:hover {
 
 .c-main-footer .c-main-footer__social {
   margin: auto;
-  @include media(">xlarge") {
```

Comment:
you might have to change this to >medium to override the styles in VFooter, did you try that?

Reply:
I think this was just a mix up on my end -- I  think this is just gothamist?